src/signalingprotocol/SignalingProtocol.d.ts

```diff
@@ -66,7 +66,7 @@ export class SdkSignalFrame implements ISdkSignalFrame {
 
     /**
      * Constructs a new SdkSignalFrame.
-     * @param [properties] Properties to set
+     * @param properties Properties to set
```

Comment:
is this change manual ? SignalingProtocol.d.ts is auto-generated through script/generate-signaling-protocol, maybe we should update the script ?

Reply:
It is a manual change.

The protobufjs version used in that script is outdated, as is its dependency on jsdoc — and neither of them produce the correct output.

generate-signaling-protocol isn't run automatically.

I will make a note to write a postprocessing script as a follow-up to make it generate correct output.